apps/marketing/src/components/(marketing)/widget.tsx

```diff
@@ -226,7 +226,9 @@ export const Widget = ({ className, children, ...props }: WidgetProps) => {
                           type="button"
                           className="bg-primary h-full w-14 rounded"
                           disabled={!field.value || !!errors.email?.message}
-                          onClick={() => onNextStepClick()}
+                          onClick={() => {
+                            if (step === 'EMAIL') onNextStepClick();
```

Comment:
No single line if statements please, you can do `step === 'EMAIL' && onNextStepClick();` if you want to be concise.

Reply:
Thanks for the Review. Done